src/platform/silabs/multi-ota/OTAFactoryDataProcessor.h

```diff
@@ -76,6 +74,9 @@ class OTAFactoryDataProcessor : public OTATlvProcessor
     OTAFactoryPayload mPayload;
     OTADataAccumulator mAccumulator;
     uint8_t * mFactoryData = nullptr;
+
+protected:
+    uint32_t GetAccumulatorSize() const override { return mLength; }
```

Comment:
Nit: Size vs Length. We should keep a uniform naming to keep it clear. Either Length or Size is ok but pick one

Reply:
I have renamed the method to 'GetAccumulatorLength'